### PR TITLE
Add FAQ questions related to LogStash and Fluent

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Signed-off-by: Naarcha-AWS <naarcha@amazon.com>

### Description
Fixes part of issue from the Documentation Project, https://github.com/opensearch-project/documentation-website/issues/467
 
### Issues Resolved
- https://github.com/opensearch-project/documentation-website/issues/467

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _faqs/02-05-what-is-opensearch-data-prepper.markdown [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+question: What is OpenSearch Data Prepper?
+category: Plugins
```
Comment (stockholmux) [inline _faqs/02-05-what-is-opensearch-data-prepper.markdown:3]: Data Prepper isn't a plugin

File: _faqs/02-06-what-is-fluentbit-and-fluentd.markdown [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+question: What is Fluentd and Fluent Bit?
+category: Plugins
+---
+
+[Fluentd](https://calyptia.com/download/) is a data collector for log data collection, processing, and forwarding. It’s written in Ruby and supports over 500 plugins including data sources, data output, parsers, formatters, and filters.
+
+[Fluent Bit](https://calyptia.com/download/) is an agent that collects data from different sources, enriches them with filters, and sends them to multiple destinations. It’s designed with performance in mind, meaning it is optimized for high throughput and low CPU and memory usage. It’s written in C and has an architecture that supports more than 70 plugins for inputs, filters, and outputs.
```
Comment (stockholmux) [inline _faqs/02-06-what-is-fluentbit-and-fluentd.markdown:8]: can we have more specific download locations?

File: _faqs/02-06-what-is-fluentbit-and-fluentd.markdown [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+Both Fluentd and Fluent Bit have OpenSearch plugins.
```
Comment (stockholmux) [inline _faqs/02-06-what-is-fluentbit-and-fluentd.markdown:10]: This is confusing. It sounds like they have OpenSearch plugins (like alerting, ISM, etc.) not plugins that enable OpenSearch on Fluentd and Fluent Bit.

File: _faqs/01-20-what-tools-do-you-recommend-for-log-and-metrics-collection.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+question: What tools do you recommend for log and metrics collection?
+category: General
+---
+
+OpenSearch is supported by a range of tools like Beats, Fluentd, Fluent Bit, and OpenTelemetry Collector. OpenSearch believes in multiple options, but will focus effort on improving Data Prepper, Fluentd, and FluentBit support going forward. OpenSearch customers who are using Beats <= 7.12.x as an agent tool, and considering open source alternatives, should migrate to Fluent Bit >= 1.9 or Open Telemetry Collector. Beats version >= 7.13 does not support OpenSearch.
```
Comment (stockholmux) [inline _faqs/01-20-what-tools-do-you-recommend-for-log-and-metrics-collection.md:6]: Don't use the term 'customers' - this is open source software.

I'm also not a fan of OpenSearch believing things :) OpenSearch doesn't have a brain.

File: _faqs/01-21-what-tools-do-you-recommend-for-log-aggregation.markdown [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
+question: What tools do you recommend for log aggregation?
+category: General
+---
+
+OpenSearch is supported by a range of tools like Data Prepper, Fluentd, Logstash and Kafka. OpenSearch believes in multiple open source options and will focus on improving the Data Prepper, Fluentd, and Kakfa support going forward.
```
Comment (stockholmux) [inline _faqs/01-21-what-tools-do-you-recommend-for-log-aggregation.markdown:6]: oxford comma after logstash.

**Review by stockholmux — CHANGES_REQUESTED** [review]
Some confusing pieces and a little grammar.

File: _faqs/02-06-what-is-fluentbit-and-fluentd.markdown (hunk 3 above)
Comment (Naarcha-AWS) [reply _faqs/02-06-what-is-fluentbit-and-fluentd.markdown:10]: I'll remove that sentence for now and loop back around when we have better instructions on implementing Fluent unto OpenSearch

**Comment by Naarcha-AWS** [comment]
@stockholmux: Do you know if there's any config adjustments I need to make to add these new questions to the FAQ? It looks fairly automated but want to make sure before we merge.

**Review by stockholmux — APPROVED** [review]
LGTM

**Comment by stockholmux** [comment]
@Naarcha-AWS No, it's all automatic.

**Pull request merged**